src/components/InputBook.js

```diff
@@ -36,29 +36,28 @@ const InputBook = () => {
     }
   };
   return (
-    <div>
-      <form>
-        <label htmlFor="test">
-          ADD NEW BOOK
-          <input type="text" name="title" placeholder="Book title" value={title || ''} onChange={(e) => setTitle(e.target.value)} />
-          <input type="text" name="author" placeholder="Author" value={author || ''} onChange={(e) => setAuthor(e.target.value)} />
-          <select type="text" name="category" value={category} onChange={(e) => setCategory(e.target.value)} required>
-            <option value="" disabled selected>
-              select Category
-            </option>
-            <option>Fiction</option>
-            <option>Horror</option>
-            <option>Romance</option>
-            <option>Science</option>
-            <option>NonFiction</option>
-          </select>
-        </label>
+    <>
+      <hr className="input-line" />
+      <h3 className="input-header"> ADD NEW BOOK</h3>
+      <form className="input-form">
+        <input type="text" name="title" placeholder="Book title" value={title || ''} onChange={(e) => setTitle(e.target.value)} className="input-title" />
+        <input type="text" name="author" placeholder="Author" value={author || ''} onChange={(e) => setAuthor(e.target.value)} className="input-author" />
```

Comment:
- [ ] While you make a great effort 😃I see that you need to decrease the padding of the input to make it like the zeplin design as you see in your design the padding from both sides top and the bottom is greater than the inputs zeplin, so it will be better 😊 if you decreasing the padding for the inputs to make it nice like the original design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://github.com/Jeddaa/bookstore/assets/83360636/503ca707-7962-444a-a333-af1f4c050521)   |    ![image](https://github.com/Jeddaa/bookstore/assets/83360636/50ae67b1-4405-4ff0-8e26-6a20a1ef3418)   |